DQMOffline/RecoB/plugins/PrimaryVertexMonitor.cc

```diff
@@ -63,8 +63,8 @@ void PrimaryVertexMonitor::bookHistograms(DQMStore::IBooker& iBooker, edm::Run c
 
   //   xPos = iBooker.book1D ("xPos","x Coordinate" ,100, -0.1, 0.1);
 
-  nbvtx = iBooker.book1D("vtxNbr", "Reconstructed Vertices in Event", 80, -0.5, 79.5);
-  nbgvtx = iBooker.book1D("goodvtxNbr", "Reconstructed Good Vertices in Event", 80, -0.5, 79.5);
+  nbvtx = iBooker.book1D("vtxNbr", "Reconstructed Vertices in Event", 150, -0.5, 149.5);
+  nbgvtx = iBooker.book1D("goodvtxNbr", "Reconstructed Good Vertices in Event", 150, -0.5, 149.5);
```

Comment:
since you're touching it, perhaps you can make it more future proof, by using the era modifiers 

https://github.com/cms-sw/cmssw/blob/1471374377d3423894ee8a8d29b95041c9cfeb38/DQMOffline/RecoB/python/PrimaryVertexMonitor_cff.py#L32-L38

and put larger values also for Phase-2.

Reply:
Sure, I hadn't noticed the problem  was there also for Phase2. Let me fix it.

Reply:
I think I will take the chance to update few other parameters that are off.

Reply:
@AdrianoDee ping.